..F..FFFF.....FF...FFFFFF...FFF.F..FFFF...........
======================================================================
FAIL: runTest (util.TestRst_10_LiteralBlock3LossyRst-MpeWriterTestCase)
TestRst_10_LiteralBlock3; Lossy doctree comparison; Writer 'rst-mpe'
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/srv/project-mpe/docutils-ext/test/util.py", line 228, in runTest
    self._test_writer(writer_class(), lossy=lossy)
  File "/srv/project-mpe/docutils-ext/test/util.py", line 135, in _test_writer
    self.__test_writer(writer, lossy=lossy)
  File "/srv/project-mpe/docutils-ext/test/util.py", line 210, in __test_writer
    diff )
AssertionError: pxml tree mismatch 
 on </srv/project-mpe/docutils-ext/var/test-rst.10.literal-block-3.rst>

--- 

+++ 

@@ -5,8 +5,11 @@

         or quoted text is a literal block.
     <literal_block xml:space="preserve">
         Whitespace, newlines, blank lines, and
-        all kinds of markup (like *this* or
-        \this) is preserved by literal blocks.
+        all kinds of markup (like 
+        <emphasis>
+            this
+         or
+        this) is preserved by literal blocks.
         
         The paragraph containing only '::'
         will be omitted from the result.

======================================================================
FAIL: runTest (util.TestRst_13_TableLossyRst-MpeWriterTestCase)
TestRst_13_Table; Lossy doctree comparison; Writer 'rst-mpe'
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/srv/project-mpe/docutils-ext/test/util.py", line 228, in runTest
    self._test_writer(writer_class(), lossy=lossy)
  File "/srv/project-mpe/docutils-ext/test/util.py", line 135, in _test_writer
    self.__test_writer(writer, lossy=lossy)
  File "/srv/project-mpe/docutils-ext/test/util.py", line 186, in __test_writer
    ("on <%s>" % self.DOC_FILE ))
AssertionError: Empty generated file. on </srv/project-mpe/docutils-ext/var/test-rst.13.table.rst>

======================================================================
FAIL: runTest (util.TestRst_13_Table2LossyRst-MpeWriterTestCase)
TestRst_13_Table2; Lossy doctree comparison; Writer 'rst-mpe'
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/srv/project-mpe/docutils-ext/test/util.py", line 228, in runTest
    self._test_writer(writer_class(), lossy=lossy)
  File "/srv/project-mpe/docutils-ext/test/util.py", line 135, in _test_writer
    self.__test_writer(writer, lossy=lossy)
  File "/srv/project-mpe/docutils-ext/test/util.py", line 179, in __test_writer
    writer=writer)['whole']#writer_name='dotmpe-rst')
  File "/srv/project-mpe/docutils-ext/lib/docutils/core.py", line 433, in publish_parts
    enable_exit_status=enable_exit_status)
  File "/srv/project-mpe/docutils-ext/lib/docutils/core.py", line 647, in publish_programmatically
    output = pub.publish(enable_exit_status=enable_exit_status)
  File "/srv/project-mpe/docutils-ext/lib/docutils/core.py", line 211, in publish
    output = self.writer.write(self.document, self.destination)
  File "/srv/project-mpe/docutils-ext/lib/docutils/writers/__init__.py", line 77, in write
    self.translate()
  File "/srv/project-mpe/docutils-ext/dotmpe/du/ext/writer/rst/__init__.py", line 41, in translate
    self.document.walkabout(visitor)
  File "/srv/project-mpe/docutils-ext/lib/docutils/nodes.py", line 173, in walkabout
    if child.walkabout(visitor):
  File "/srv/project-mpe/docutils-ext/lib/docutils/nodes.py", line 173, in walkabout
    if child.walkabout(visitor):
  File "/srv/project-mpe/docutils-ext/lib/docutils/nodes.py", line 173, in walkabout
    if child.walkabout(visitor):
  File "/srv/project-mpe/docutils-ext/lib/docutils/nodes.py", line 173, in walkabout
    if child.walkabout(visitor):
  File "/srv/project-mpe/docutils-ext/lib/docutils/nodes.py", line 173, in walkabout
    if child.walkabout(visitor):
  File "/srv/project-mpe/docutils-ext/lib/docutils/nodes.py", line 173, in walkabout
    if child.walkabout(visitor):
  File "/srv/project-mpe/docutils-ext/lib/docutils/nodes.py", line 165, in walkabout
    visitor.dispatch_visit(self)
  File "/srv/project-mpe/docutils-ext/lib/docutils/nodes.py", line 1604, in dispatch_visit
    return method(node)
  File "/srv/project-mpe/docutils-ext/dotmpe/du/ext/writer/rst/__init__.py", line 114, in append
    assert hasattr(self.stack.formatter, visitor_name), visitor_name
AssertionError: visit_rubric

======================================================================
FAIL: runTest (util.TestRst_13_Table3LossyRst-MpeWriterTestCase)
TestRst_13_Table3; Lossy doctree comparison; Writer 'rst-mpe'
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/srv/project-mpe/docutils-ext/test/util.py", line 228, in runTest
    self._test_writer(writer_class(), lossy=lossy)
  File "/srv/project-mpe/docutils-ext/test/util.py", line 135, in _test_writer
    self.__test_writer(writer, lossy=lossy)
  File "/srv/project-mpe/docutils-ext/test/util.py", line 186, in __test_writer
    ("on <%s>" % self.DOC_FILE ))
AssertionError: Empty generated file. on </srv/project-mpe/docutils-ext/var/test-rst.13.table-3.rst>

======================================================================
FAIL: runTest (util.TestRst_13_Table4LossyRst-MpeWriterTestCase)
TestRst_13_Table4; Lossy doctree comparison; Writer 'rst-mpe'
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/srv/project-mpe/docutils-ext/test/util.py", line 228, in runTest
    self._test_writer(writer_class(), lossy=lossy)
  File "/srv/project-mpe/docutils-ext/test/util.py", line 135, in _test_writer
    self.__test_writer(writer, lossy=lossy)
  File "/srv/project-mpe/docutils-ext/test/util.py", line 186, in __test_writer
    ("on <%s>" % self.DOC_FILE ))
AssertionError: Empty generated file. on </srv/project-mpe/docutils-ext/var/test-rst.13.table-4.rst>

======================================================================
FAIL: runTest (util.TestRst_17_Tables1LossyRst-MpeWriterTestCase)
TestRst_17_Tables1; Lossy doctree comparison; Writer 'rst-mpe'
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/srv/project-mpe/docutils-ext/test/util.py", line 228, in runTest
    self._test_writer(writer_class(), lossy=lossy)
  File "/srv/project-mpe/docutils-ext/test/util.py", line 135, in _test_writer
    self.__test_writer(writer, lossy=lossy)
  File "/srv/project-mpe/docutils-ext/test/util.py", line 186, in __test_writer
    ("on <%s>" % self.DOC_FILE ))
AssertionError: Empty generated file. on </srv/project-mpe/docutils-ext/var/test-rst.17.tables-1.rst>

======================================================================
FAIL: runTest (util.TestRst_17_Tables2LossyRst-MpeWriterTestCase)
TestRst_17_Tables2; Lossy doctree comparison; Writer 'rst-mpe'
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/srv/project-mpe/docutils-ext/test/util.py", line 228, in runTest
    self._test_writer(writer_class(), lossy=lossy)
  File "/srv/project-mpe/docutils-ext/test/util.py", line 135, in _test_writer
    self.__test_writer(writer, lossy=lossy)
  File "/srv/project-mpe/docutils-ext/test/util.py", line 186, in __test_writer
    ("on <%s>" % self.DOC_FILE ))
AssertionError: Empty generated file. on </srv/project-mpe/docutils-ext/var/test-rst.17.tables-2.rst>

======================================================================
FAIL: runTest (util.TestRst_1_Document2LossyRst-MpeWriterTestCase)
TestRst_1_Document2; Lossy doctree comparison; Writer 'rst-mpe'
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/srv/project-mpe/docutils-ext/test/util.py", line 228, in runTest
    self._test_writer(writer_class(), lossy=lossy)
  File "/srv/project-mpe/docutils-ext/test/util.py", line 135, in _test_writer
    self.__test_writer(writer, lossy=lossy)
  File "/srv/project-mpe/docutils-ext/test/util.py", line 210, in __test_writer
    diff )
AssertionError: pxml tree mismatch 
 on </srv/project-mpe/docutils-ext/var/test-rst.1.document-2.rst>

--- 

+++ 

@@ -3,11 +3,4 @@

         Test Document
     <paragraph>
         Some paragraph.
-    <section ids="section" names="section">
-        <title>
-            Section
-        <paragraph>
-            Some paragraph.
-        <comment xml:space="preserve">
-            comment
 

======================================================================
FAIL: runTest (util.TestRst_1_Document3LossyRst-MpeWriterTestCase)
TestRst_1_Document3; Lossy doctree comparison; Writer 'rst-mpe'
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/srv/project-mpe/docutils-ext/test/util.py", line 228, in runTest
    self._test_writer(writer_class(), lossy=lossy)
  File "/srv/project-mpe/docutils-ext/test/util.py", line 135, in _test_writer
    self.__test_writer(writer, lossy=lossy)
  File "/srv/project-mpe/docutils-ext/test/util.py", line 210, in __test_writer
    diff )
AssertionError: pxml tree mismatch 
 on </srv/project-mpe/docutils-ext/var/test-rst.1.document-3.rst>

--- 

+++ 

@@ -5,68 +5,4 @@

         Some paragraph.
     <comment xml:space="preserve">
         Comment
-    <section ids="section" names="section">
-        <title>
-            Section
-        <paragraph>
-            Some paragraph.
-        <field_list>
-            <field>
-                <field_name>
-                    field
-                <field_body>
-                    <paragraph>
-                        value
-            <field>
-                <field_name>
-                    field
-                <field_body>
-                    <paragraph>
-                        value
-        <enumerated_list enumtype="arabic" prefix="" suffix=".">
-            <list_item>
-                <paragraph>
-                    List item 1
-            <list_item>
-                <paragraph>
-                    List item 2
-        <enumerated_list enumtype="loweralpha" prefix="" suffix=".">
-            <list_item>
-                <paragraph>
-                    List item a.
-            <list_item>
-                <paragraph>
-                    List item b.
-        <enumerated_list enumtype="upperalpha" prefix="" suffix=".">
-            <list_item>
-                <paragraph>
-                    List item A.
-            <list_item>
-                <paragraph>
-                    List item B.
-        <enumerated_list enumtype="upperroman" prefix="" suffix=".">
-            <list_item>
-                <paragraph>
-                    List item I.
-            <list_item>
-                <paragraph>
-                    List item II.
-        <definition_list>
-            <definition_list_item>
-                <term>
-                    Term
-                <definition>
-                    <paragraph>
-                        Definition
-            <definition_list_item>
-                <term>
-                    Term
-                <definition>
-                    <paragraph>
-                        Definition
-        <section ids="subsection" names="subsection">
-            <title>
-                Subsection
-            <paragraph>
-                Some paragraph in 3.1
 

======================================================================
FAIL: runTest (util.TestRst_1_Document4LossyRst-MpeWriterTestCase)
TestRst_1_Document4; Lossy doctree comparison; Writer 'rst-mpe'
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/srv/project-mpe/docutils-ext/test/util.py", line 228, in runTest
    self._test_writer(writer_class(), lossy=lossy)
  File "/srv/project-mpe/docutils-ext/test/util.py", line 135, in _test_writer
    self.__test_writer(writer, lossy=lossy)
  File "/srv/project-mpe/docutils-ext/test/util.py", line 179, in __test_writer
    writer=writer)['whole']#writer_name='dotmpe-rst')
  File "/srv/project-mpe/docutils-ext/lib/docutils/core.py", line 433, in publish_parts
    enable_exit_status=enable_exit_status)
  File "/srv/project-mpe/docutils-ext/lib/docutils/core.py", line 647, in publish_programmatically
    output = pub.publish(enable_exit_status=enable_exit_status)
  File "/srv/project-mpe/docutils-ext/lib/docutils/core.py", line 211, in publish
    output = self.writer.write(self.document, self.destination)
  File "/srv/project-mpe/docutils-ext/lib/docutils/writers/__init__.py", line 77, in write
    self.translate()
  File "/srv/project-mpe/docutils-ext/dotmpe/du/ext/writer/rst/__init__.py", line 41, in translate
    self.document.walkabout(visitor)
  File "/srv/project-mpe/docutils-ext/lib/docutils/nodes.py", line 173, in walkabout
    if child.walkabout(visitor):
  File "/srv/project-mpe/docutils-ext/lib/docutils/nodes.py", line 173, in walkabout
    if child.walkabout(visitor):
  File "/srv/project-mpe/docutils-ext/lib/docutils/nodes.py", line 165, in walkabout
    visitor.dispatch_visit(self)
  File "/srv/project-mpe/docutils-ext/lib/docutils/nodes.py", line 1604, in dispatch_visit
    return method(node)
  File "/srv/project-mpe/docutils-ext/dotmpe/du/ext/writer/rst/__init__.py", line 114, in append
    assert hasattr(self.stack.formatter, visitor_name), visitor_name
AssertionError: visit_author

======================================================================
FAIL: runTest (util.TestRst_1_Document5_FullRstDemoLossyRst-MpeWriterTestCase)
TestRst_1_Document5_FullRstDemo; Lossy doctree comparison; Writer 'rst-mpe'
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/srv/project-mpe/docutils-ext/test/util.py", line 228, in runTest
    self._test_writer(writer_class(), lossy=lossy)
  File "/srv/project-mpe/docutils-ext/test/util.py", line 135, in _test_writer
    self.__test_writer(writer, lossy=lossy)
  File "/srv/project-mpe/docutils-ext/test/util.py", line 179, in __test_writer
    writer=writer)['whole']#writer_name='dotmpe-rst')
  File "/srv/project-mpe/docutils-ext/lib/docutils/core.py", line 433, in publish_parts
    enable_exit_status=enable_exit_status)
  File "/srv/project-mpe/docutils-ext/lib/docutils/core.py", line 647, in publish_programmatically
    output = pub.publish(enable_exit_status=enable_exit_status)
  File "/srv/project-mpe/docutils-ext/lib/docutils/core.py", line 211, in publish
    output = self.writer.write(self.document, self.destination)
  File "/srv/project-mpe/docutils-ext/lib/docutils/writers/__init__.py", line 77, in write
    self.translate()
  File "/srv/project-mpe/docutils-ext/dotmpe/du/ext/writer/rst/__init__.py", line 41, in translate
    self.document.walkabout(visitor)
  File "/srv/project-mpe/docutils-ext/lib/docutils/nodes.py", line 173, in walkabout
    if child.walkabout(visitor):
  File "/srv/project-mpe/docutils-ext/lib/docutils/nodes.py", line 173, in walkabout
    if child.walkabout(visitor):
  File "/srv/project-mpe/docutils-ext/lib/docutils/nodes.py", line 165, in walkabout
    visitor.dispatch_visit(self)
  File "/srv/project-mpe/docutils-ext/lib/docutils/nodes.py", line 1604, in dispatch_visit
    return method(node)
  File "/srv/project-mpe/docutils-ext/dotmpe/du/ext/writer/rst/__init__.py", line 114, in append
    assert hasattr(self.stack.formatter, visitor_name), visitor_name
AssertionError: visit_author

======================================================================
FAIL: runTest (util.TestRst_1_Document6LossyRst-MpeWriterTestCase)
TestRst_1_Document6; Lossy doctree comparison; Writer 'rst-mpe'
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/srv/project-mpe/docutils-ext/test/util.py", line 228, in runTest
    self._test_writer(writer_class(), lossy=lossy)
  File "/srv/project-mpe/docutils-ext/test/util.py", line 135, in _test_writer
    self.__test_writer(writer, lossy=lossy)
  File "/srv/project-mpe/docutils-ext/test/util.py", line 179, in __test_writer
    writer=writer)['whole']#writer_name='dotmpe-rst')
  File "/srv/project-mpe/docutils-ext/lib/docutils/core.py", line 433, in publish_parts
    enable_exit_status=enable_exit_status)
  File "/srv/project-mpe/docutils-ext/lib/docutils/core.py", line 647, in publish_programmatically
    output = pub.publish(enable_exit_status=enable_exit_status)
  File "/srv/project-mpe/docutils-ext/lib/docutils/core.py", line 211, in publish
    output = self.writer.write(self.document, self.destination)
  File "/srv/project-mpe/docutils-ext/lib/docutils/writers/__init__.py", line 77, in write
    self.translate()
  File "/srv/project-mpe/docutils-ext/dotmpe/du/ext/writer/rst/__init__.py", line 41, in translate
    self.document.walkabout(visitor)
  File "/srv/project-mpe/docutils-ext/lib/docutils/nodes.py", line 173, in walkabout
    if child.walkabout(visitor):
  File "/srv/project-mpe/docutils-ext/lib/docutils/nodes.py", line 173, in walkabout
    if child.walkabout(visitor):
  File "/srv/project-mpe/docutils-ext/lib/docutils/nodes.py", line 165, in walkabout
    visitor.dispatch_visit(self)
  File "/srv/project-mpe/docutils-ext/lib/docutils/nodes.py", line 1604, in dispatch_visit
    return method(node)
  File "/srv/project-mpe/docutils-ext/dotmpe/du/ext/writer/rst/__init__.py", line 114, in append
    assert hasattr(self.stack.formatter, visitor_name), visitor_name
AssertionError: visit_author

======================================================================
FAIL: runTest (util.TestRst_1_Document7LossyRst-MpeWriterTestCase)
TestRst_1_Document7; Lossy doctree comparison; Writer 'rst-mpe'
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/srv/project-mpe/docutils-ext/test/util.py", line 228, in runTest
    self._test_writer(writer_class(), lossy=lossy)
  File "/srv/project-mpe/docutils-ext/test/util.py", line 135, in _test_writer
    self.__test_writer(writer, lossy=lossy)
  File "/srv/project-mpe/docutils-ext/test/util.py", line 210, in __test_writer
    diff )
AssertionError: pxml tree mismatch 
 on </srv/project-mpe/docutils-ext/var/test-rst.1.document-7.rst>

--- 

+++ 

@@ -11,16 +11,12 @@

                 The fanatics going straight forward and openly, the politicians by the surer mode of zigzag.
         <attribution>
             Burke.
-    <enumerated_list enumtype="arabic" prefix="" start="2" suffix=".">
-        <list_item>
-            <paragraph>
-                (Arch.) A molding running in a zigzag line; a chevron, or series of chevrons.
-        <list_item>
-            <paragraph>
-                (Fort.) See Boyau. 
-                <footnote_reference auto="1" ids="id1" refid="id2">
-                    1
-    <footnote auto="1" backrefs="id1" ids="id2" names="1">
+    <paragraph>
+        2. (Arch.) A molding running in a zigzag line; a chevron, or series of chevrons.
+        2. (Fort.) See Boyau. 
+        <footnote_reference ids="id1" refid="id2">
+            1
+    <footnote auto="1" ids="id2" names="1">
         <label>
             1
         <paragraph>

======================================================================
FAIL: runTest (util.TestRst_21_Transition1LossyRst-MpeWriterTestCase)
TestRst_21_Transition1; Lossy doctree comparison; Writer 'rst-mpe'
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/srv/project-mpe/docutils-ext/test/util.py", line 228, in runTest
    self._test_writer(writer_class(), lossy=lossy)
  File "/srv/project-mpe/docutils-ext/test/util.py", line 135, in _test_writer
    self.__test_writer(writer, lossy=lossy)
  File "/srv/project-mpe/docutils-ext/test/util.py", line 203, in __test_writer
    warnings)
AssertionError: Error re-parsing generated file
 on </srv/project-mpe/docutils-ext/var/test-rst.21.transition-1.rst>

/srv/project-mpe/docutils-ext/var/test-rst.21.transition-1.rst:5: (ERROR/3) Document may not end with a transition.


======================================================================
FAIL: runTest (util.TestRst_22_DocinfoLossyRst-MpeWriterTestCase)
TestRst_22_Docinfo; Lossy doctree comparison; Writer 'rst-mpe'
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/srv/project-mpe/docutils-ext/test/util.py", line 228, in runTest
    self._test_writer(writer_class(), lossy=lossy)
  File "/srv/project-mpe/docutils-ext/test/util.py", line 135, in _test_writer
    self.__test_writer(writer, lossy=lossy)
  File "/srv/project-mpe/docutils-ext/test/util.py", line 179, in __test_writer
    writer=writer)['whole']#writer_name='dotmpe-rst')
  File "/srv/project-mpe/docutils-ext/lib/docutils/core.py", line 433, in publish_parts
    enable_exit_status=enable_exit_status)
  File "/srv/project-mpe/docutils-ext/lib/docutils/core.py", line 647, in publish_programmatically
    output = pub.publish(enable_exit_status=enable_exit_status)
  File "/srv/project-mpe/docutils-ext/lib/docutils/core.py", line 211, in publish
    output = self.writer.write(self.document, self.destination)
  File "/srv/project-mpe/docutils-ext/lib/docutils/writers/__init__.py", line 77, in write
    self.translate()
  File "/srv/project-mpe/docutils-ext/dotmpe/du/ext/writer/rst/__init__.py", line 41, in translate
    self.document.walkabout(visitor)
  File "/srv/project-mpe/docutils-ext/lib/docutils/nodes.py", line 173, in walkabout
    if child.walkabout(visitor):
  File "/srv/project-mpe/docutils-ext/lib/docutils/nodes.py", line 173, in walkabout
    if child.walkabout(visitor):
  File "/srv/project-mpe/docutils-ext/lib/docutils/nodes.py", line 165, in walkabout
    visitor.dispatch_visit(self)
  File "/srv/project-mpe/docutils-ext/lib/docutils/nodes.py", line 1604, in dispatch_visit
    return method(node)
  File "/srv/project-mpe/docutils-ext/dotmpe/du/ext/writer/rst/__init__.py", line 114, in append
    assert hasattr(self.stack.formatter, visitor_name), visitor_name
AssertionError: visit_author

======================================================================
FAIL: runTest (util.TestRst_23_OptionList1LossyRst-MpeWriterTestCase)
TestRst_23_OptionList1; Lossy doctree comparison; Writer 'rst-mpe'
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/srv/project-mpe/docutils-ext/test/util.py", line 228, in runTest
    self._test_writer(writer_class(), lossy=lossy)
  File "/srv/project-mpe/docutils-ext/test/util.py", line 135, in _test_writer
    self.__test_writer(writer, lossy=lossy)
  File "/srv/project-mpe/docutils-ext/test/util.py", line 162, in __test_writer
    warnings)
AssertionError: Corrupt test source file: on </srv/project-mpe/docutils-ext/var/test-rst.23.option-list-1.rst>
/srv/project-mpe/docutils-ext/var/test-rst.23.option-list-1.rst:2: (WARNING/2) Title underline too short.

\ufeffOption Lists
------------


======================================================================
FAIL: runTest (util.TestRst_2_Title1LossyRst-MpeWriterTestCase)
TestRst_2_Title1; Lossy doctree comparison; Writer 'rst-mpe'
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/srv/project-mpe/docutils-ext/test/util.py", line 228, in runTest
    self._test_writer(writer_class(), lossy=lossy)
  File "/srv/project-mpe/docutils-ext/test/util.py", line 135, in _test_writer
    self.__test_writer(writer, lossy=lossy)
  File "/srv/project-mpe/docutils-ext/test/util.py", line 210, in __test_writer
    diff )
AssertionError: pxml tree mismatch 
 on </srv/project-mpe/docutils-ext/var/test-rst.2.title-1.rst>

--- 

+++ 

@@ -3,16 +3,4 @@

         Title
     <subtitle dupnames="title" ids="id1">
         Title
-    <section dupnames="title" ids="id2">
-        <title>
-            Title
-        <section dupnames="title" ids="id3">
-            <title>
-                Title
-            <section dupnames="title" ids="id4">
-                <title>
-                    Title
-                <section dupnames="title" ids="id5">
-                    <title>
-                        Title
 

======================================================================
FAIL: runTest (util.TestRst_5_Inline1LossyRst-MpeWriterTestCase)
TestRst_5_Inline1; Lossy doctree comparison; Writer 'rst-mpe'
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/srv/project-mpe/docutils-ext/test/util.py", line 228, in runTest
    self._test_writer(writer_class(), lossy=lossy)
  File "/srv/project-mpe/docutils-ext/test/util.py", line 135, in _test_writer
    self.__test_writer(writer, lossy=lossy)
  File "/srv/project-mpe/docutils-ext/test/util.py", line 203, in __test_writer
    warnings)
AssertionError: Error re-parsing generated file
 on </srv/project-mpe/docutils-ext/var/test-rst.5.inline-1.rst>

/srv/project-mpe/docutils-ext/var/test-rst.5.inline-1.rst:9: (ERROR/3) Indirect hyperlink target "reference" (id="reference") refers to target "target", which does not exist.


======================================================================
FAIL: runTest (util.TestRst_5_Inline2LossyRst-MpeWriterTestCase)
TestRst_5_Inline2; Lossy doctree comparison; Writer 'rst-mpe'
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/srv/project-mpe/docutils-ext/test/util.py", line 228, in runTest
    self._test_writer(writer_class(), lossy=lossy)
  File "/srv/project-mpe/docutils-ext/test/util.py", line 135, in _test_writer
    self.__test_writer(writer, lossy=lossy)
  File "/srv/project-mpe/docutils-ext/test/util.py", line 203, in __test_writer
    warnings)
AssertionError: Error re-parsing generated file
 on </srv/project-mpe/docutils-ext/var/test-rst.5.inline-2.rst>

/srv/project-mpe/docutils-ext/var/test-rst.5.inline-2.rst:5: (WARNING/2) Explicit markup ends without a blank line; unexpected unindent.
/srv/project-mpe/docutils-ext/var/test-rst.5.inline-2.rst:7: (WARNING/2) malformed hyperlink target.
/srv/project-mpe/docutils-ext/var/test-rst.5.inline-2.rst:25: (WARNING/2) Explicit markup ends without a blank line; unexpected unindent.
/srv/project-mpe/docutils-ext/var/test-rst.5.inline-2.rst:30: (WARNING/2) Bullet list ends without a blank line; unexpected unindent.
/srv/project-mpe/docutils-ext/var/test-rst.5.inline-2.rst:31: (WARNING/2) Inline interpreted text or phrase reference start-string without end-string.
/srv/project-mpe/docutils-ext/var/test-rst.5.inline-2.rst:33: (WARNING/2) Definition list ends without a blank line; unexpected unindent.
/srv/project-mpe/docutils-ext/var/test-rst.5.inline-2.rst:33: (WARNING/2) Inline interpreted text or phrase reference start-string without end-string.
/srv/project-mpe/docutils-ext/var/test-rst.5.inline-2.rst:35: (WARNING/2) malformed hyperlink target.
/srv/project-mpe/docutils-ext/var/test-rst.5.inline-2.rst:37: (WARNING/2) malformed hyperlink target.
/srv/project-mpe/docutils-ext/var/test-rst.5.inline-2.rst:: (ERROR/3) Anonymous hyperlink mismatch: 1 references but 0 targets.
See "backrefs" attribute for IDs.
/srv/project-mpe/docutils-ext/var/test-rst.5.inline-2.rst:5: (ERROR/3) Unknown target name: "anonymous inline reference".


======================================================================
FAIL: runTest (util.TestRst_5_Inline3LossyRst-MpeWriterTestCase)
TestRst_5_Inline3; Lossy doctree comparison; Writer 'rst-mpe'
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/srv/project-mpe/docutils-ext/test/util.py", line 228, in runTest
    self._test_writer(writer_class(), lossy=lossy)
  File "/srv/project-mpe/docutils-ext/test/util.py", line 135, in _test_writer
    self.__test_writer(writer, lossy=lossy)
  File "/srv/project-mpe/docutils-ext/test/util.py", line 203, in __test_writer
    warnings)
AssertionError: Error re-parsing generated file
 on </srv/project-mpe/docutils-ext/var/test-rst.5.inline-3.rst>

/srv/project-mpe/docutils-ext/var/test-rst.5.inline-3.rst:9: (ERROR/3) Unknown interpreted text role "role".
/srv/project-mpe/docutils-ext/var/test-rst.5.inline-3.rst:11: (ERROR/3) Unknown interpreted text role "role".
/srv/project-mpe/docutils-ext/var/test-rst.5.inline-3.rst:13: (ERROR/3) Unknown interpreted text role "role".


======================================================================
FAIL: runTest (util.TestRst_5_Inline4LossyRst-MpeWriterTestCase)
TestRst_5_Inline4; Lossy doctree comparison; Writer 'rst-mpe'
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/srv/project-mpe/docutils-ext/test/util.py", line 228, in runTest
    self._test_writer(writer_class(), lossy=lossy)
  File "/srv/project-mpe/docutils-ext/test/util.py", line 135, in _test_writer
    self.__test_writer(writer, lossy=lossy)
  File "/srv/project-mpe/docutils-ext/test/util.py", line 203, in __test_writer
    warnings)
AssertionError: Error re-parsing generated file
 on </srv/project-mpe/docutils-ext/var/test-rst.5.inline-4.rst>

/srv/project-mpe/docutils-ext/var/test-rst.5.inline-4.rst:13: (WARNING/2) Explicit markup ends without a blank line; unexpected unindent.
/srv/project-mpe/docutils-ext/var/test-rst.5.inline-4.rst:3: (ERROR/3) Too many symbol footnote references: only 0 corresponding footnotes available.
/srv/project-mpe/docutils-ext/var/test-rst.5.inline-4.rst:3: (ERROR/3) Unknown target name: "python".
/srv/project-mpe/docutils-ext/var/test-rst.5.inline-4.rst:3: (ERROR/3) Unknown target name: "example".
/srv/project-mpe/docutils-ext/var/test-rst.5.inline-4.rst:3: (ERROR/3) Unknown target name: "python web site".
/srv/project-mpe/docutils-ext/var/test-rst.5.inline-4.rst:3: (ERROR/3) Unknown target name: "1".
/srv/project-mpe/docutils-ext/var/test-rst.5.inline-4.rst:3: (ERROR/3) Unknown target name: "3".
/srv/project-mpe/docutils-ext/var/test-rst.5.inline-4.rst:3: (ERROR/3) Unknown target name: "2".
/srv/project-mpe/docutils-ext/var/test-rst.5.inline-4.rst:3: (ERROR/3) Unknown target name: "cit2002".
/srv/project-mpe/docutils-ext/var/test-rst.5.inline-4.rst:13: (ERROR/3) Unknown target name: "targets".
/srv/project-mpe/docutils-ext/var/test-rst.5.inline-4.rst:17: (ERROR/3) Unknown target name: "pep 287".
/srv/project-mpe/docutils-ext/var/test-rst.5.inline-4.rst:17: (ERROR/3) Unknown target name: "rfc 2822".


----------------------------------------------------------------------
Ran 50 tests in 3.358s

FAILED (failures=21)
